docs/running-node-ansible/vn.md

```diff
@@ -0,0 +1,167 @@
+```
+</details>
+
+#### 3. (Optional) If you are joining a long-running network, enable `statesync` or use one of the options in [running-node-in-existing-network.md](../advanced/running-node-in-existing-network.md)
+[`deployment/ansible/roles/configure/vars/validator.yml`]
+
+```yaml
+config:
+...
+  statesync:
+    enable: true
+    rpc_servers: "http(s):<node1-IP>:26657, ..."
+    trust_height: <trust-height>
+    trust_hash: "<trust-hash>"
+...
+```
+</details>
+
+> **_NOTE:_**  You should provide at least 2 addresses for `rpc_servers`. It can be 2 identical addresses
+
+You can use the following command to obtain `<trust-height>` and `<trust-hash>` of your network
+
+```bash
+curl -s http(s)://<host>:<port>/commit | jq "{height: .result.signed_header.header.height, hash: .result.signed_header.commit.block_id.hash}"
+```
+<details>
+<summary>Example for Testnet 2.0</summary>
+
+```bash
+curl -s https://on.test-net.dcl.csa-iot.org:26657/commit | jq "{height: .result.signed_header.header.height, hash: .result.signed_header.commit.block_id.hash}"
+```
+</details>
+
+- `<host>` - RPC endpoint host of the network being joined
+- `<port>` - RPC endpoint port of the network being joined
+
+> **_NOTE:_** State sync is not attempted if the node has any local state (LastBlockHeight > 0)
+
+## Run ansible (local machine)
+#### 1. Verify that all the configuration parameters from the previous section are correct
+#### 2. Run ansible
+```bash
+ansible-playbook -i ./deployment/ansible/inventory  -u <target-host-ssh-user> ./deployment/ansible/deploy.yml
+```
+- `<target-host-ssh-username>` - target host ssh user
+- Ansible provisioning can take several minutes depending on number of nodes being provisioned
+
+## Deployment Verification (target machine)
+#### 1. Switch to cosmovisor user
+```
+sudo su -s /bin/bash cosmovisor
+```
+
+#### 2. Query status
+```
+dcld status
+```
+
+## Make your node a Validator (target machine)
+
+#### 1. Switch to cosmovisor user
+```
+sudo su -s /bin/bash cosmovisor
+```
+
+#### 2. Generate NodeAdmin keys
+
+```bash
+dcld keys add "<admin-account-name>" 2>&1 | tee "<admin-account-name>.dclkey.data"
+```
+
+IMPORTANT keep generated data (especially the mnemonic) securely.
+
+#### 3. Share generated address and pubkey with the community
+
+address and pubkey can be found using
+
+```bash
+dcld keys show --output text "<admin-account-name>"
+```
+
+#### 4. Wait until your NodeAdmin key is proposed and approved by the quorum of trustees
+
+- Make sure the Node Admin account is proposed by a Trustee (usually CSA). The account will appear in the "Accounts" / "All Proposed Accounts" tab in https://testnet.iotledger.io/accounts.
```

Comment:
Please mention, that the link `https://testnet.iotledger.io` is for the Test Net only.